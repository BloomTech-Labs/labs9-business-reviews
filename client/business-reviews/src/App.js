import React, { Component } from 'react';
import styled from 'styled-components'
import FeaturedReviews from './components/FeaturedReviews'
import PopularBusinesses from './components/PopularBusinesses'
import PopularReviewers from './components/PopularReviewers'
import Footer from './components/Footer'

const Container = styled.div`
  margin: 0 auto;
  display: flex;
  flex-flow: row wrap;
  justify-content: center;
  width: 100%;
`;

class App extends Component {
  render() {
    return (
      <div className="App">
        <Container>
          <FeaturedReviews />
          <PopularBusinesses />
          <PopularReviewers />
          <Footer />
        </Container>
      </div>
    );
  }
}

export default App;
