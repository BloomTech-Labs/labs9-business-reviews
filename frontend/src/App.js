import React from 'react';
import styled from 'styled-components';
import { Route, Switch } from 'react-router-dom';
import LandingPage from './components/LandingPage';
import UserProfile from './components/UserProfile';
import BillingPage from './components/BillingPage';
import Authenticate from './components/Authenticate';

const StyledApp = styled.div`
  margin: 0 auto;
  max-width: 1200px;
`;

function App() {
  return (
    <StyledApp className="App">
      <Switch>
        <Route exact path="/" component={LandingPage} />
        <Route path="/user" component={UserProfile} />
        <Route path="/billing" component={BillingPage} />
        <Route path="/authenticate" component={Authenticate} />
      </Switch>
    </StyledApp>
  );
}

export default App;
