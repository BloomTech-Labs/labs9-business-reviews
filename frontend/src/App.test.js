import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import App from './App';



describe('<App />', () => {
  it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
  });
  it('should match snapshot', () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});