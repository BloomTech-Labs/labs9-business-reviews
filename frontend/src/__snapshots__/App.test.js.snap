// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> matches snapshot 1`] = `
<div
  className="App sc-eNQAEJ HLyMB"
>
  <div
    className="sc-bdVaJa fFQAZv"
  >
    <h2
      className="menu__logo--logo"
    >
      Bonafide.biz
    </h2>
    <div
      className="menu__menuItems"
    >
      \\
      <a>
        add a review
      </a>
      <a>
        categories
      </a>
      <a>
        top rated
      </a>
    </div>
    <div
      className="menu__user"
    >
      <a
        className="menu__user--text"
      >
        Sign Up
      </a>
      <a
        className="menu__user--text"
      >
        Sign In
      </a>
    </div>
  </div>
  <div
    className="sc-dnqmqq byOlbM"
  >
    <div
      className="sc-bwzfXH fXLiti"
    >
      <form>
        <input
          placeholder="Search..."
        />
      </form>
      <div
        className="button-container"
      >
        <button
          className="button"
        >
          Review
        </button>
        <button
          className="button"
        >
          Search
        </button>
      </div>
    </div>
    <div>
      <div
        className="sc-bZQynM hcklJe"
      >
        <h1>
          Featured Reviews
        </h1>
        <div
          className="sc-gzVnrw bgMTQM"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/sushi"
          />
          <h1>
            ⭐⭐⭐⭐⭐
          </h1>
          <p>
            Ramiace
          </p>
          <a
            href="/user/@ktkdsfghs"
            onClick={[Function]}
          >
            <p>
              @ktkdsfghs
            </p>
          </a>
        </div>
        <div
          className="sc-gzVnrw bgMTQM"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/storefront"
          />
          <h1>
            ⭐⭐⭐
          </h1>
          <p>
            Bartramiaceae
          </p>
          <a
            href="/user/@hthrtujsjk"
            onClick={[Function]}
          >
            <p>
              @hthrtujsjk
            </p>
          </a>
        </div>
        <div
          className="sc-gzVnrw bgMTQM"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/pizza"
          />
          <h1>
            ⭐⭐⭐⭐
          </h1>
          <p>
            Umbilicariaceae
          </p>
          <a
            href="/user/@conelieusfsdf"
            onClick={[Function]}
          >
            <p>
              @conelieusfsdf
            </p>
          </a>
        </div>
        <div
          className="sc-gzVnrw bgMTQM"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/shop"
          />
          <h1>
            ⭐⭐⭐⭐⭐
          </h1>
          <p>
            Asteraceae
          </p>
          <a
            href="/user/@asdfadsfa"
            onClick={[Function]}
          >
            <p>
              @asdfadsfa
            </p>
          </a>
        </div>
      </div>
      <div
        className="sc-ifAKCX doBRCR"
      >
        <h1>
          Popular Businesses
        </h1>
        <div
          className="sc-EHOje ckZctU"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/mcdonalds"
          />
          <h1>
            ⭐⭐⭐⭐⭐
          </h1>
          <p>
            McDonald's
          </p>
        </div>
        <div
          className="sc-EHOje ckZctU"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/kmart"
          />
          <h1>
            ⭐⭐⭐
          </h1>
          <p>
            KMart
          </p>
        </div>
        <div
          className="sc-EHOje ckZctU"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/walmart"
          />
          <h1>
            ⭐⭐⭐⭐
          </h1>
          <p>
            Walmart
          </p>
        </div>
        <div
          className="sc-EHOje ckZctU"
        >
          <img
            alt="reviewed business"
            src="https://loremflickr.com/200/200/target"
          />
          <h1>
            ⭐⭐⭐⭐⭐
          </h1>
          <p>
            Target
          </p>
        </div>
      </div>
      <div
        className="sc-htpNat IohGC"
      >
        <h1>
          Popular Reviewers
        </h1>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/man"
          />
          <p>
            ManGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/dog"
          />
          <p>
            DogGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/woman"
          />
          <p>
            WomanGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/kid"
          />
          <p>
            KidGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/cat"
          />
          <p>
            CatGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/person"
          />
          <p>
            PersonGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/human"
          />
          <p>
            HumanGuy
          </p>
        </div>
        <div
          className="sc-bxivhb kXvwFe"
        >
          <img
            alt="reviewers"
            src="https://loremflickr.com/200/200/bird"
          />
          <p>
            BirdGuy
          </p>
        </div>
      </div>
    </div>
    <div
      className="sc-htoDjs dkzXYj"
    >
      <a
        href="www.google.com"
      >
        Meet the Team
      </a>
    </div>
  </div>
</div>
`;
